ogm/contribute.adoc

```diff
@@ -95,12 +100,9 @@ If you just want to build the documentation only,
 run it from the [filename]+hibernate-ogm-documentation/manual+ subdirectory.
 ====
 
-=== How to contribute code effectively
+=== How to contribute the code effectively
```

Comment:
"How to contribute" is the main title. The code is not the only way you can contribute.
"How to contribute code effectively" is correct, I'll remove the "the".